auto_submit/lib/service/validation_service.dart

```diff
@@ -349,3 +362,59 @@ class ProcessMergeResult {
   bool result = false;
   String? message;
 }
+
+/// Function signature that will be executed with retries.
+typedef RetryHandler = Function();
+
+/// Runs the internal processMerge with retries.
+Future<void> _runProcessMergeWithRetries(RetryHandler retryHandler, {RetryOptions? retryOptions}) {
+  final RetryOptions r = retryOptions ??
+      const RetryOptions(
+        delayFactor: Duration(milliseconds: Config.backOfMultiplier),
+        maxDelay: Duration(seconds: Config.maxDelaySeconds),
+        maxAttempts: Config.backoffAttempts,
+      );
```

Comment:
Seems `retryOptions` is always not null.

Reply:
Fixed.